### PR TITLE
Added more detailed example for running Sydney in docker

### PR DESCRIPTION
**Pull request by uchu**
As we found out that some parameters like a node key is critical to have public key in telemetry, I've decided to provide a full example of running a validator node here, not only in docs.

As people maybe be confused and will start using these minimal example, rather than checking docs.

Maybe we can add an extrac sections. Any ideas are welcome.

### CODE REVIEW

**Comment by humfecki** [comment]
@uchu  should we link how to create the key page somewhere here? it's clear that docker expecting a keyfile from command but maybe its not very clear that key needs to be pre-created. 

Its already described in docs. [key generation](https://docs.ggxchain.io/developer-documentation/keys/node-create-keys.html) in order to create keys,

**Comment by uchu** [comment]
> @uchu should we link how to create the key page somewhere here? it's clear that docker is expecting a keyfile from the command but maybe its not very clear that key needs to be pre-created.
> 
> Its already described in docs. [key generation](https://docs.ggxchain.io/developer-documentation/keys/node-create-keys.html) in order to create keys,


@humfecki 
I just put a notice that they need to create keys, before the link to the documentation. Is it enough,  wdyt?

**Comment by humfecki** [comment]
@uchu sounds good, direct link could be helpful but docs is there to read and link might change in future.

File: README.md [hunk 1]
```diff
@@ -55,23 +52,42 @@ It takes around 20 mins to complete with this suggested requirements, exponentia
...
 docker build -f Dockerfile.sydney -t ggxchain-node:sydney .
 
-docker run \
-    -it \
-    --rm \
-    --name ggx-local-node \
-    -u $(id -g):$(id -u) \
-    -p 30333:30333 \
+mkdir -p data-sydney
+
```
Comment (obsessed-cake) [inline README.md:64]:
```suggestion
cp your_node_key data-sydney/node.key
# or create a new node key
subkey generate-node-key > data-sydney/node.key
```

**Pull request merged**